plugins/pitchfork.py

```diff
@@ -17,7 +18,11 @@ def pitchfork(self, msg, arg):
         """
         match = re.match(r'@?([\w-]+)(?:\s+(?:down\s+)?to\s+(.+))?$',
                          arg)
-        if match:
+        labhub_util = LabHub("bot")
+        inviter = msg.frm.nick
+
+        if match and labhub_util.Teams[labhub_util.GH_ORG_NAME +
```

Comment:
It should not be in a single if statement, it should be nested if-else statements. give some message if command user is not a developer.